server/api.proto

```diff
@@ -261,18 +261,26 @@ message Friend {
 }
 
 message TFriendAdd {
+  message UserIds {
```

Comment:
In other places, like `TMatchJoin` we used the approach in this PR. Changing it to the way you mentioned in the comment will make the API inconsistent unless we change those places as well. 

The server-side complexity is a little bit more, but not too hard.

I can be persuaded in either direction :)